src/nix-store/nix-store.cc

```diff
@@ -141,6 +143,7 @@ static void opRealise(Strings opFlags, Strings opArgs)
         toDerivedPaths(paths),
         willBuild, willSubstitute, unknown, downloadSize, narSize);
 
+    /* filter out unkown paths from `paths` */
```

Comment:
```suggestion
    /* Filter out unknown paths from `paths`. */
```

And similarly capitalize/punctuate the other comments.

Reply:
Done.